R/get_xronos.R

```diff
@@ -0,0 +1,36 @@
+#' @rdname db_getter_backend
+#' @export
+get_xronos <- function(db_url = get_db_url("xronos")) {
+
+  check_connection_to_url(db_url)
+
+  resp<-httr::GET(db_url)
+
+  jsonRespParsed<-httr::content(resp,as="parsed")
+
+  xronos_data <- jsonRespParsed %>% tidyr::tibble() %>%
+    tidyr::unnest_wider(1) %>%
+    tidyr::unnest_wider(measurement)
```

Comment:
These procedures take a significant amount of time and could be avoided with a simple flat file download.... :sweat_smile: